src/main/java/io/kestra/plugin/linkedin/CommentTrigger.java

```diff
@@ -0,0 +1,322 @@
+
+            Output output = Output.builder()
+                .postUrn(latest.getPostUrn())
+                .commentId(latest.getCommentId())
+                .commentUrn(latest.getCommentUrn())
+                .commentText(latest.getCommentText())
+                .actorUrn(latest.getActorUrn())
+                .agentUrn(latest.getAgentUrn())
+                .createdTime(latest.getCreatedTime())
+                .newCommentsCount(newComments.size())
+                .allNewComments(newComments)
+                .build();
+
+            Execution execution = TriggerService.generateExecution(this, conditionContext, context, output);
+            return Optional.of(execution);
+
+        } catch (Exception e) {
+            runContext.logger().error("Error checking for new comments", e);
+            throw new RuntimeException("Failed to check for new comments: " + e.getMessage(), e);
+        }
+    }
+
+    private CommentData parseCommentData(String postUrn, JsonObject commentObj, Instant lastCheckTime) {
+        if (!commentObj.has("created") || !commentObj.has("message")) {
+            return null;
+        }
+
+        JsonObject created = commentObj.getAsJsonObject("created");
+        if (!created.has("time")) {
+            return null;
+        }
+
+        long createdTimeMs = created.get("time").getAsLong();
+        Instant createdTime = Instant.ofEpochMilli(createdTimeMs);
+
+        // Only include comments created after the last check
+        if (!createdTime.isAfter(lastCheckTime)) {
+            return null;
+        }
+
+        JsonObject message = commentObj.getAsJsonObject("message");
+        String commentText = message.has("text") ? message.get("text").getAsString() : "";
+
+        String commentId = commentObj.has("id") ? commentObj.get("id").getAsString() : null;
+        String commentUrn = commentObj.has("commentUrn") ? commentObj.get("commentUrn").getAsString() : null;
+        String actorUrn = commentObj.has("actor") ? commentObj.get("actor").getAsString() : null;
+        String agentUrn = commentObj.has("agent") ? commentObj.get("agent").getAsString() : null;
+
+        return CommentData.builder()
+            .postUrn(postUrn)
+            .commentId(commentId)
+            .commentUrn(commentUrn)
+            .commentText(commentText)
+            .actorUrn(actorUrn)
+            .agentUrn(agentUrn)
+            .createdTime(createdTime)
+            .build();
+    }
+
+    private HttpRequestFactory createLinkedinHttpRequestFactory(String accessToken) throws Exception {
+        com.google.api.client.auth.oauth2.Credential credential = 
+            new com.google.api.client.auth.oauth2.Credential(
+                com.google.api.client.auth.oauth2.BearerToken.authorizationHeaderAccessMethod()
+            );
+        credential.setAccessToken(accessToken);
+
+        return new com.google.api.client.http.javanet.NetHttpTransport().createRequestFactory(credential);
```

Comment:
No need to put the `com.google.api.client.http.javanet.` here if you properly import the class